README.md

```diff
@@ -87,6 +87,8 @@ class. *rich-argparse* defines the following styles:
 }
 ```
 
+Any color supported by [Rich](https://github.com/Textualize/rich) works here ([this list of color names is currently supplied](https://rich.readthedocs.io/en/latest/appendix/colors.html#appendix-colors)).
```

Comment:
This is not useful if you add the link to the style page as suggested above. Please remove.